module-bsp/board/rt1051/bellpx/clock_config.cpp

```diff
@@ -168,7 +168,7 @@ void BOARD_BootClockRUN(void)
     CLOCK_SetMux(kCLOCK_Usdhc1Mux, 0); // CSCMR1  (16) 0 - PLL2_PFD2, 1 - PLL2_PFD0
 
     /* Set USDHC2_PODF. */
-    CLOCK_SetDiv(kCLOCK_Usdhc2Div, 2); // CSCDR1
+    CLOCK_SetDiv(kCLOCK_Usdhc2Div, 3); // CSCDR1 Div by 3 to get ~82MHz clock
```

Comment:
It seems that it is in reality divided by 4, as the effective division value is equal to `div + 1` (ignore the 99MHz value, I just opened some example project from MCUXpresso that has different clock config than MuditaOS)

![image](https://github.com/mudita/MuditaOS/assets/26345399/354d70f0-2ca6-48f4-a2e1-54fa18da48fd)

Reply:
You're right ;) The problem is that the MCUXpresso counts clk div from 1..n, but we use a range from 0..n-1 in the code. 
I'm going to modify the comment in the code like: `CLOCK_SetDiv(kCLOCK_Usdhc2Div, 3); // CSCDR1 Div by 4 (3 + 1) to get ~82MHz clock`. Is it ok?

Reply:
Sure! I just wanted to ensure you are aware of this difference :smile: